README.md

```diff
@@ -45,6 +45,7 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 ## Index
 
 - [Android Development](#android-development)
+- [Angular](#angular)
```

Comment:
Angular section has already been added to the index, delete this as well.
make sure to sync your local repo before contributing to ignore merge conflicts.